static/ca_bydate.js

```diff
@@ -173,9 +173,11 @@ function redoFilter() {
   table.clearFilter();
   if (countySel.value) {
     table.addFilter("county", "=", countySel.value);
+    setSubHeader = document.getElementById("sub-header").innerHTML = countySel.value;
```

Comment:
looks like this is unused. i'm gonna remove it from the version it put into master. let me know if i'm missing something.

Reply:
See comment below